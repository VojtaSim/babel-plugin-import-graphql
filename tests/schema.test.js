import path from 'path'
import { transformWithPlugin } from './util'

describe.each([{ runtime: true }, {}])('plugin options = %j', opts => {
  describe('simple schema document', () => {
    test('inlines as raw source text', () => {
      const { code } = transformWithPlugin('./fixtures/schema/simple/default.js', opts)
      let _graphqlTag, _graphqlTag2, schema
      eval(code)
      expect(schema).toMatchSnapshot()
    })
  })
})
