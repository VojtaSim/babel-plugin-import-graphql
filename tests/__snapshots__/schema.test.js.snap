// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`plugin options = {"runtime":true} simple schema document inlines as raw source text 1`] = `
"schema {
  query: Query
}

type Query {
  test: String
}
"
`;

exports[`plugin options = {} simple schema document inlines as raw source text 1`] = `
"schema {
  query: Query
}

type Query {
  test: String
}
"
`;
